src/Common/Command/AbstractCommand.php

```diff
@@ -97,9 +97,17 @@ protected function getOptionBool(string $name): bool
         return true === $this->input->getOption($name);
     }
 
-    protected function getOptionInt(string $name): int
+    protected function getOptionInt(string $name, ?int $default = null): int
     {
-        return \intval($this->input->getOption($name));
+        if (null !== $option = $this->input->getOption($name)) {
+            return \intval($this->input->getOption($name));
+        }
+
+        if ($default) {
```

Comment:
```suggestion
        if (null !== $default) {
```
Otherwise if $default is set to 0 you'll throw an error